nni/retiarii/execution/python.py

```diff
@@ -3,23 +3,25 @@
 from ..graph import Evaluator, Model
 from ..integration_api import receive_trial_parameters
 from ..utils import ContextStack, import_, get_importable_name
-from .base import BaseExecutionEngine
+from .base import BaseExecutionEngine, extract_visual_hp
 
 
 class PythonGraphData:
     def __init__(self, class_name: str, init_parameters: Dict[str, Any],
-                 mutation: Dict[str, Any], evaluator: Evaluator) -> None:
+                 mutation: Dict[str, Any], evaluator: Evaluator, visual_hp: dict = None) -> None:
         self.class_name = class_name
         self.init_parameters = init_parameters
         self.mutation = mutation
         self.evaluator = evaluator
+        self.visual_hp = visual_hp
 
     def dump(self) -> dict:
         return {
             'class_name': self.class_name,
             'init_parameters': self.init_parameters,
             'mutation': self.mutation,
-            'evaluator': self.evaluator
+            'evaluator': self.evaluator,
+            '_visual_hyper_params_': self.visual_hp
```

Comment:
I don't think `visual_hyper_params` is so special that it needs to be quoted with underline.

Reply:
I think that depends on whether `_visual_hyper_params_` is designed to be Retiarii only.
In HPO scenario all other stuffs are meant for tuning algorithm, but this field is for web UI.

Reply:
some other stuffs are meant for trial. in hpo, those stuffs are visualizable, but in retiarii they are not. that is why we need a new field. "_visual_hyper_params_" is special, because it delivers redundant data